packages/core/stdlib/math/main.browse

```diff
@@ -1,12 +1,30 @@
+#* @scope { Standard Math functions }
+#  @name { Math }
+#  TODO: This is a bit hacky. If the @name is called below the import it will be attributed to E rather than to the scope.
```

Comment:
That's not hacky at all. I it actually makes sense that comments relating to the entire file SHOULD be right at the top of the file, before all the imports

I think you can remove this TODO